src/Python/qsharp-core/qsharp/__init__.py

```diff
@@ -55,6 +55,10 @@ def compile(code : str) -> Union[QSharpCallable, List[QSharpCallable]]:
     :returns: A list of callables compiled from `code`, or a callable if exactly
         one callable is found.
     """
+    compiled = client.compile(code)
+    if compiled is None:
```

Comment:
I think we discussed this but please remind me - Does `client.compile` raise any compilation errors or are we suppressing them?

Reply:
Neither exactly; client.compile prints errors to stderr and returns `None` on compilation failure. We could consider capturing and raising as Python errors, but if so I'd like to defer that to a future PR since this one is purely a bugfix for existing behavior.

Reply:
Sounds good, thanks!